extension/src/popup/components/accountHistory/HistoryItem/index.tsx

```diff
@@ -89,7 +89,9 @@ export const HistoryItem = ({
             ? `${recipientLabel} ${operationAssetCode}`
             : "Transaction",
           operationText: isPayment
-            ? `${paymentDifference}${amount} ${operationAssetCode}`
+            ? `${paymentDifference}${new BigNumber(
+                amount,
+              )} ${operationAssetCode}`
```

Comment:
do you need a .toString() here for the BigNumber?

I think you shouldn't but iirc I had to add it under certain scenarios otherwise would get a "react cant render an object" runtime error, so just making sure

Reply:
Good callout - this didn't cause an issue because I believe a JS template string automatically coerces anything within the `${}` into a string

Reply:
It actually looks like `.toString()` wasn't need up above either